metadata.yaml

```diff
@@ -2,17 +2,14 @@ name: jenkins
 summary: Jenkins Continuous Integration Server
 maintainer: Free Ekanayaka <free.ekanayaka@canonical.com>
 description: |
```

Comment:
Change the | to > while you are here, as this description will render better with > normalized whitespace rather than | preformatted text.